codecarbon/core/co2_signal.py

```diff
@@ -0,0 +1,35 @@
+import requests
+
+from codecarbon.core.units import EmissionsPerKwh, Energy
+from codecarbon.external.geography import GeoMetadata
+
+URL = "https://api.co2signal.com/v1/latest"
+CO2_SIGNAL_API_TOKEN = None
```

Comment:
It would be better to pass the token to the `is_available` and `get_emissions` functions or create an intermediary object to store it